delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/utils/NonEmptySetImplicits.scala

```diff
@@ -0,0 +1,16 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.utils
+
+import cats.data.NonEmptySet
+
+object NonEmptySetImplicits {
```

Comment:
Rename to `NonEmptySetSyntax` and move it to `ch.epfl.bluebrain.nexus.delta.sdk.syntax`, add it to the package object where it will meet some others